src/serializer/visitors.js

```diff
@@ -116,6 +135,23 @@ export let ClosureRefReplacer = {
     }
   },
 
+  // TODO: handle FunctionDeclaration
+  FunctionExpression(path: BabelTraversePath, state: ClosureRefReplacerState) {
+    if (t.isProgram((path: any).parentPath.parentPath.node)) {
+      // Skip root function itself.
+      // This assumes the root function is wrapped with: t.file(t.program([t.expressionStatement(rootFunction).
+      return;
+    }
+
+    const functionExpression: BabelNodeFunctionExpression = path.node;
+    const functionTag: number = (functionExpression.body: any).uniqueTag;
```

Comment:
Instead of these unsafe cast that end with :any, consider defining an interface `HasUniqueTag` with `uniqueTag`, and add a cast to that interface. (This applies to all other uses of `uniqueTag` as well.) That way, at least any spelling error of `uniqueTag` would be caught.

Reply:
Makes sense, will do.